tests/BookStore/Integration/Doctrine/DoctrineBookRepositoryTest.php

```diff
@@ -202,6 +203,8 @@ public function testCount(): void
             $repository->save($book);
         }
 
+        static::getContainer()->get(EntityManagerInterface::class)->clear();
```

Comment:
Maybe it makes more sense to put the clear in `testOfId` instead, WDYT?

Reply:
Works for me, done!